core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/workspacemanager/HttpWorkspaceManagerDAOSpec.scala

```diff
@@ -294,26 +294,31 @@ class HttpWorkspaceManagerDAOSpec
 
     val billingProjectId = "billing-project-namespace";
 
-    val expectedRequest = new CreateWorkspaceRequestBody()
-      .id(testData.azureWorkspace.workspaceIdAsUUID)
-      .displayName(testData.azureWorkspace.name)
-      .spendProfile(testData.azureBillingProfile.getId.toString)
-      .stage(WorkspaceStageModel.MC_WORKSPACE)
-      .applicationIds(Seq("exampleApp").asJava)
-      .policies(policyInputs)
-      .projectOwnerGroupId(billingProjectId)
-
     wsmDao.createWorkspaceWithSpendProfile(
       testData.azureWorkspace.workspaceIdAsUUID,
       testData.azureWorkspace.name,
       testData.azureBillingProfile.getId.toString,
       billingProjectId,
       Seq("exampleApp"),
+      CloudPlatform.AZURE,
       Some(policyInputs),
       testContext
     )
 
-    verify(workspaceApi).createWorkspace(expectedRequest)
+    val createWorkspaceArgumentCaptor = captor[CreateWorkspaceV2Request]
+    verify(workspaceApi).createWorkspaceV2(createWorkspaceArgumentCaptor.capture)
+    // V2-specific values
+    createWorkspaceArgumentCaptor.getValue.getJobControl shouldNot be(null)
```

Comment:
Since the jobId is randomly created when the method is called, we don't have a way to compare it.